elf_loader/src/elf_loader.c

```diff
@@ -522,9 +522,9 @@ elf_loader_relocate(struct elf_loader_t *loader)
  * @return const char * version of ei_class
  */
 const char *
-ei_class_to_str(unsigned char ei_class)
+ei_class_to_str(unsigned char _ei_class)
```

Comment:
Please drop the "_ei" for these so that we are consistent. Also, please update the doxygen comments to reflect the changes.